crates/store/src/db/mod.rs

```diff
@@ -48,6 +49,13 @@ pub struct NullifierInfo {
     pub block_num: BlockNumber,
 }
 
+#[derive(Debug, PartialEq)]
+pub struct TransactionInfo {
```

Comment:
Should we rename this into `TransactionSummary` as well?

Reply:
sure, done!